addons/hide-flyout/userscript.js

```diff
@@ -38,6 +48,10 @@ export default async function ({ addon, global, console }) {
     lockDisplay.classList.add("sa-flyoutClose");
     lockDisplay.style.transitionDuration = `${speed}s`;
     setTimeout(() => Blockly.getMainWorkspace().recordCachedAreas(), speed * 1000);
+    if (toggleSetting === "category") {
+      for (let category of document.querySelectorAll(".scratchCategoryMenuItem.categorySelected"))
```

Comment:
This will be slow, but it's probably fine

Reply:
This part can mostly be rewritten to use Blockly's data